src/components/Application-Header/JobsAppBar.tsx

```diff
@@ -1,30 +1,94 @@
 import * as React from 'react';
-
+import { withStyles, WithStyles, createStyles } from '@material-ui/core';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
-
+import AccessibilityNewIcon from '@material-ui/icons/AccessibilityNew';
+import Avatar from '@material-ui/core/Avatar';
+import IconButton from '@material-ui/core/IconButton';
+import StarIcon from '@material-ui/icons/Star';
+import Tooltip from '@material-ui/core/Tooltip';
 import { User } from '../../types/userTypes';
 
-interface AppBarData {
+const styles = () => createStyles({
+    root: {
+        backgroundColor: 'gray'
+    },
+    toolbar: { 
+        display: 'flex',
+        justifyContent: 'space-between'
+    },
+    avatar: {
+        backgroundColor: 'rgb(87,87,87)',
+        margin: '0 5px'
```

Comment:
Why exactly 5 if I may ask? 😄

Reply:
just to get some space between the objects.